Src/Support/GoogleApis/Apis/Requests/HttpRequestMessageExtenstions.cs

```diff
@@ -19,7 +19,11 @@
 using System.Net.Http.Headers;
 using System.Text;
 
+#if (NET45 || NETSTANDARD)

```

Comment:
I'm surprised this is the *only* difference... is the API for Ionic.Zlib exactly the same as for System.IO.Compression?